README.md

```diff
@@ -12,30 +12,30 @@ For Amazon EKS clusters, the Mountpoint for Amazon S3 CSI driver is also availab
-| v1.7.0         | public.ecr.aws/mountpoint-s3-csi-driver/aws-mountpoint-s3-csi-driver:v1.7.0                       |
-| v1.6.0         | public.ecr.aws/mountpoint-s3-csi-driver/aws-mountpoint-s3-csi-driver:v1.6.0                       |
-| v1.5.1         | public.ecr.aws/mountpoint-s3-csi-driver/aws-mountpoint-s3-csi-driver:v1.5.1                       |
-| v1.4.0         | public.ecr.aws/mountpoint-s3-csi-driver/aws-mountpoint-s3-csi-driver:v1.4.0                       |
-| v1.3.1         | public.ecr.aws/mountpoint-s3-csi-driver/aws-mountpoint-s3-csi-driver:v1.3.1                       |
-| v1.3.0         | public.ecr.aws/mountpoint-s3-csi-driver/aws-mountpoint-s3-csi-driver:v1.3.0                       |
-| v1.2.0         | public.ecr.aws/mountpoint-s3-csi-driver/aws-mountpoint-s3-csi-driver:v1.2.0                       |
-| v1.1.0         | public.ecr.aws/mountpoint-s3-csi-driver/aws-mountpoint-s3-csi-driver:v1.1.0                       |
-| v1.0.0         | public.ecr.aws/mountpoint-s3-csi-driver/aws-mountpoint-s3-csi-driver:v1.0.0                       |
+| v1.12.0        | public.ecr.aws/mountpoint-s3-csi-driver/scality-mountpoint-s3-csi-driver:v1.12.0                      |
+| v1.11.0        | public.ecr.aws/mountpoint-s3-csi-driver/scality-mountpoint-s3-csi-driver:v1.11.0                      |
+| v1.10.0        | public.ecr.aws/mountpoint-s3-csi-driver/scality-mountpoint-s3-csi-driver:v1.10.0                      |
+| v1.9.0         | public.ecr.aws/mountpoint-s3-csi-driver/scality-mountpoint-s3-csi-driver:v1.9.0                       |
+| v1.8.1         | public.ecr.aws/mountpoint-s3-csi-driver/scality-mountpoint-s3-csi-driver:v1.8.1                       |
+| v1.8.0         | public.ecr.aws/mountpoint-s3-csi-driver/scality-mountpoint-s3-csi-driver:v1.8.0                       |
+| v1.7.0         | public.ecr.aws/mountpoint-s3-csi-driver/scality-mountpoint-s3-csi-driver:v1.7.0                       |
+| v1.6.0         | public.ecr.aws/mountpoint-s3-csi-driver/scality-mountpoint-s3-csi-driver:v1.6.0                       |
+| v1.5.1         | public.ecr.aws/mountpoint-s3-csi-driver/scality-mountpoint-s3-csi-driver:v1.5.1                       |
+| v1.4.0         | public.ecr.aws/mountpoint-s3-csi-driver/scality-mountpoint-s3-csi-driver:v1.4.0                       |
+| v1.3.1         | public.ecr.aws/mountpoint-s3-csi-driver/scality-mountpoint-s3-csi-driver:v1.3.1                       |
+| v1.3.0         | public.ecr.aws/mountpoint-s3-csi-driver/scality-mountpoint-s3-csi-driver:v1.3.0                       |
+| v1.2.0         | public.ecr.aws/mountpoint-s3-csi-driver/scality-mountpoint-s3-csi-driver:v1.2.0                       |
+| v1.1.0         | public.ecr.aws/mountpoint-s3-csi-driver/scality-mountpoint-s3-csi-driver:v1.1.0                       |
+| v1.0.0         | public.ecr.aws/mountpoint-s3-csi-driver/scality-mountpoint-s3-csi-driver:v1.0.0                       |
```

Comment:
We don't  push into aws ecr's repository and we don't have those versions. We should completely replace this with our github registry ?

Reply:
I deleted these in the next PR: https://github.com/scality/mountpoint-s3-csi-driver/pull/29